checkpointctl.go

```diff
@@ -89,6 +90,13 @@ func setupInspect() *cobra.Command {
 		false,
 		"Display an overview of mounts used in the container checkpoint",
 	)
+	flags.Uint32VarP(
+		&pID,
+		"pid",
+		"p",
+		0,
+		"Specify a particular process in the container checkpoint",
+	)
```

Comment:
Should we automatically set `psTree = true` when the `--pid` options is used?

Reply:
:thinking: I don't think we should, because the `--pid` option is not exclusive to `--ps-tree`. It works with the child flags too, like `--files`. If `--pid` is passed without any flag that generates a process tree, we should do nothing, IMO.

Reply:
Sure with `--ps-tree` and `--files` we generate a process tree, while `--pid <PID>` can be used as a filter. It would be easier to use when the default behaviour is to show a process tree without explicitly specifying `--ps-tree`.
```
checkpointctl inspect --pid 5 /tmp/checkpoint.tar.gz
checkpointctl inspect --files --pid 5 /tmp/checkpoint.tar.gz
```